src/main/java/org/broadinstitute/hellbender/tools/copynumber/CondenseReadCounts.java

```diff
@@ -0,0 +1,203 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.util.BlockCompressedOutputStream;
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SimpleCountCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.SimpleCount;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@CommandLineProgramProperties(summary = "condenses read-counts in .tsv format into a new file also in .tsv format",
+        oneLineSummary = "condenses read-counts in .tsv format into a new file also in .tsv formas" ,
+        programGroup = CopyNumberProgramGroup.class)
+public class CondenseReadCounts extends CommandLineProgram {
+
+    public static final String FACTOR_LONG_NAME = "factor";
+    public static final String FACTOR_SHORT_NAME = "f";
+    public static final String EXPECTED_BIN_LENGTH_LONG_NAME = "out-bin-length";
+
+    @Argument(
+            doc = "Path to the input counts file",
+            fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME
+    )
+    private String input;
+
+    @Argument(
+            doc = "Path to the output counts file",
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME
+    )
+    private String output;
+
+    @Argument(
+            doc = "Indicates how many input read count record must be condense into each output record",
```

Comment:
I'd suggest `Indicates how many consecutive input read count records will be combined for each output record`